lib/container/ContainerPlugin.js

```diff
@@ -73,6 +74,7 @@ class ContainerPlugin {
 		}
 
 		compiler.hooks.make.tapAsync(PLUGIN_NAME, (compilation, callback) => {
+			const hooks = ModuleFederationPlugin.getCompilationHooks(compilation);
```

Comment:
These compilation hooks are used in several places in my plugin as well as in my next plugin to handle the modules in various ways. I also have runtime modules that use this as well.